GOVERNANCE.md

```diff
@@ -90,22 +89,62 @@ in terms of project direction, to potentially less experienced contributors.
 * Make decisions about the overall scope, vision and direction of the project.
 * Developing funding sources
 * Deciding how to disburse funds with consultation from Core Contributors
 
 The council may choose to delegate these responsibilities to sub-committees. If so, Council members must update this document to make the delegation clear.
 
-Note that individual council member does not have the power to unilaterally wield these responsibilities, but the council as a whole must jointly make these decisions. In other words, Council Members are first and foremost Core Contributors, but only when needed they can collectively make decisions for the health of the project.
-
-ArviZ will be holding its first election to determine its initial council in the coming weeks and this document will be updated.
+Note that individual council member does not have the power to unilaterally wield these
+responsibilities, but the council as a whole must jointly make these decisions.
+In other words, Council Members are first and foremost Core Contributors, but only
+when needed they can collectively make decisions for the health of the project.
+
+### Council Decision Making Process
+By and large we expect the decisions in ArviZ to be made _ad hoc_ and require little formal
+coordination and with the community at large. However for controversial proposals and
+new core contributors the council may need to intervene to make the final decision
+in a group vote.
+
+#### Call for a vote
+Core contributors can call for a vote to resolve a target issue they feel has
+been stale for too long and informal consensus appears unlikely. For a voting
+to be called, the target issue must be at least 2 months old.
+
+To do so, they have to open a proposal issue ticket labeled "Council Vote".
+The proposal issue should contain a link to the target issue and a proposal
+on how to resolve it. Proposals should include a statement making clear what it
+means to "agree" or to "disagree".
+
+Before voting starts, at least 3 days will be left for Core
+Contributors to raise doubts about the proposal's _phrasing_, no extra discussion will take
+place in the proposal issue. Proposal issues should be locked from creation
+to prevent attracting discussion from people not familiar with the decision
+process.
+
+#### Voting process
+
+* Each council member will vote either Yes, No, or Neutral.
+* It is recommended that all council members expose their reasons when voting.
+  "No" votes however _must_ list the reasons for disagreement. Any "no" vote
+  with no reason listed will be considered a "neutral" vote.
+* An absence of vote is considered as neutral.
+* Voting will remain open for at least 3 days.
+* For the proposal to pass, at least 60% of the council must vote Yes, and no more
+  than 20% can vote No.
+
+For decisions about the project the Council will perform it directly on the
+proposal issue.
+For decisions about people, such as Core Contributors or ejecting folks, the Council
```

Comment:
"folks" is very informal for this document. Also, what does "ejecting" mean here?

Reply:
I think ejecting here refers to the `Ejecting Core Contributors` section. Maybe we could reword as:

    For decisions about people, such as electing or ejecting Core Contributors, the Council

Reply:
Also, this should probably go in another PR but is the ejecting for core contributors only? What about single time contributors (or people with no relation to ArviZ) being disrespective in issues and breaking CoC?

Reply:
I'm fine with that change.